x-pack/plugin/esql/compute/src/main/java/org/elasticsearch/compute/operator/Driver.java

```diff
@@ -116,7 +113,7 @@ public class Driver implements Releasable, Describable {
      */
     public Driver(
         String sessionId,
-        String taskDescription,
+        String shortDescription,
```

Comment:
nit: why `shortDescription` here and `description` in `DriverProfile`?

Reply:
I named it `description` there as `driver_description` would bee to verbose as it is already in the driver object :
```
  "profile": {
    "drivers": [
      {
        "description": "final", <----
        "start_millis": 1739864633522,
```

I would name it here driver as well, but we already have such field defined for describe:
https://github.com/elastic/elasticsearch/blob/e8b01ff7c0de20396b8f375050bb2fdf195d3b80/x-pack/plugin/esql/compute/src/main/java/org/elasticsearch/compute/operator/Driver.java#L460-L463

Reply:
Sorry, I hadn't realized the name conflict with `description` below, which would've been my preferred name for the parameter here. I think it's fine as-is!